### DISCUSSION TITLE
I don't know why, but pictures on github Android app never load.

### DISCUSSION THREAD

**Discussion by uniquepersun** [community/community#67055] [Mobile]
They just show the alt text that's it, but if you do same web it works perfectly.
Please do resolve this issue..

**Comment by alcere** [comment]
Hey @unique-persun, we are sorry for the troubles here! Is this something that happens to all types of images? or is it related to a specific format? In addition, if you could share some examples of images not rendering that will incredibly helpful for the dev team to investigate the problem, thanks!

**Reply by uniquepersun** [reply]
Yeah sure, lemme just screenshot some.

**Reply by uniquepersun** [reply]
![Screenshot_20230915-162202_GitHub.png](https://github.com/community/community/assets/109142198/10e12170-a8f2-4bf4-88be-a97cd06e1251)

![Screenshot_20230915-162146_Chrome.png](https://github.com/community/community/assets/109142198/985f4da5-ce49-4ab8-895b-bf2141a58ef8)

![Screenshot_20230915-162139_Chrome.png](https://github.com/community/community/assets/109142198/a7d59a92-c3bf-4993-8ea2-1cda3bae93a4)

![Screenshot_20230915-161511_GitHub.png](https://github.com/community/community/assets/109142198/4441dd42-cf28-49de-bd2e-4a296c70a893)
👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻👆🏻
Pictures like this are perfectly loading, look on those statistics and all, even bun logo was completely loaded 

But,
Png is only black and svg showing path code.
I used GitHub desktop on my mobile as I currently don't have my desktop.

![Screenshot_20230915-161500_GitHub.png](https://github.com/community/community/assets/109142198/5559fcb5-5ea9-4195-82cb-401ccc3dbb35)

![Screenshot_20230915-161450_GitHub.png](https://github.com/community/community/assets/109142198/a139856e-7d17-4c05-8543-60048ea8195a)

![Screenshot_20230915-161522_GitHub.png](https://github.![Screenshot_20230915-161851_Chrome.png](https://github.com/community/community/assets/109142198/045888c8-ad15-49dd-a2a3-76e68b5f6c01)

85c-b945-2adb93f642c6)

![Uploading Screenshot_20230915-161851_Chrome.png …]()

![Screenshot_20230915-161423_GitHub.png](https://github.com/community/community/assets/109142198/90e4e13a-6841-41e9-8b65-c66730103b82)

![Screenshot_20230915-161857_Chrome.png](https://github.com/community/community/assets/109142198/1593ba60-b1d3-4095-9490-f8f255d48967)



I mostly able to find png images, i don't know if it with jpeg or not.

**Reply by uniquepersun** [reply]
There is a problem, I can see those screenshots here and these also are in png format, but still can't see in any repository folder or on home
